### PR TITLE
PWN-7533 - New Swap. Proxy backend

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7533
https://p2pvalidator.atlassian.net/browse/PWN-7541

## Description of Work
- removed recent_blockhash from SDK request

### CODE REVIEW

**Review by 1sakov — APPROVED** [review]

File: app/src/main/res/values/environment_config.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
     <string name="orcaApiBaseUrl">https://orca.key.app/</string>
 
-    <string name="jupiterQuoteBaseUrl">https://quote-api.jup.ag/</string>
+    <string name="jupiterQuoteBaseUrl">https://swap.keyapp.org/</string>
```
Comment (1sakov) [inline app/src/main/res/values/environment_config.xml:29]: what about the prod url? 
When we'll add it?

File: app/src/main/java/org/p2p/wallet/swap/jupiter/interactor/JupiterSwapInteractor.kt [hunk 2]
```diff
@@ -37,13 +35,14 @@ class JupiterSwapInteractor(
...
-        val latestBlockhash = rpcBlockhashRepository.getRecentBlockhash()
         val userAccount = Account(tokenKeyProvider.keyPair)
 
         val signedSwapTransaction = relaySdkFacade.signTransaction(
             transaction = swapTransaction,
             keyPair = userAccount.getEncodedKeyPair().toBase58Instance(),
-            recentBlockhash = latestBlockhash
+            // empty string because swap transaction already has recent blockhash
+            // if pass our own recent blockhash, there is an error
+            recentBlockhash = null
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/swap/jupiter/interactor/JupiterSwapInteractor.kt:45]: but you are cleaning the current blockhash by this, no?

File: app/src/main/res/values/environment_config.xml (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/res/values/environment_config.xml:29]: there no prod ulr, we'll discuss it today

File: app/src/main/java/org/p2p/wallet/swap/jupiter/interactor/JupiterSwapInteractor.kt (hunk 2 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/swap/jupiter/interactor/JupiterSwapInteractor.kt:45]: no, the current blockhash in transaction already exists from Jupiter, and our SDK overwrites it that causes error from RPC

**Review by kabramovp2p — APPROVED** [review]

**Pull request merged**